### PR TITLE
Update codeql.yml

### PR DESCRIPTION
**Pull request by XhmikosR**
@Havunen missed this in #1

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -29,7 +29,6 @@ jobs:
       - name: Initialize CodeQL
         uses: github/codeql-action/init@v3
         with:
-          config-file: ./.github/codeql/codeql-config.yml
```
Comment (Havunen) [inline .github/workflows/codeql.yml:32]: Why remove the config?

Comment (XhmikosR) [reply .github/workflows/codeql.yml:32]: Because it doesn't exist since it's not needed.

Comment (Havunen) [reply .github/workflows/codeql.yml:32]: isn't it this one? https://github.com/Havunen/detective-vue2/blob/main/.github/workflows/codeql.yml

Comment (Havunen) [reply .github/workflows/codeql.yml:32]: ah config, right, LGTM

**Pull request merged**